src/test/regress/expected/propagate_alter_role.out

```diff
@@ -0,0 +1,227 @@
+CREATE SCHEMA alter_role;
+SET citus.enable_alter_role_propagation to ON;
+SELECT run_command_on_workers($$SET citus.enable_alter_role_propagation to ON;$$);
+ run_command_on_workers  
+-------------------------
+ (localhost,57637,t,SET)
+ (localhost,57638,t,SET)
+(2 rows)
+
+CREATE ROLE alter_role_1 WITH LOGIN;
```

Comment:
it would be nice to use a role name that has some escaped characters like `"alter_role'1"` or such to test the edge cases